bots/images/scripts/fedora.setup

```diff
@@ -164,23 +172,25 @@ if ! [ "$version" -eq "$version" ] 2>/dev/null; then version=30; fi
 su builder -c "/usr/bin/mock --no-bootstrap-chroot --verbose -i $(/var/lib/testvm/build-deps.sh "$opsys $version")"
 su builder -c "/usr/bin/mock --install --verbose rpmlint"
 
-# HACK: docker falls over regularly, print its log if it does
-systemctl start docker || journalctl -u docker
-
-# our cockpit/base container is only really a thing on x86_64, just skip it on other arches
-if [ $(uname -m) = x86_64 ]; then
-    docker build -t cockpit/base /var/tmp/cockpit-base
+# HACK docker not available on f31
+# https://github.com/cockpit-project/cockpit/issues/12670
+if [ "$IMAGE" != fedora-31 ]; then
```

Comment:
Let's perhaps make this `if type docker >/dev/null 2>&1`, as long as it's not yet clear which relelases docker will end up on in the long run. Or something similar to `$HAVE_KUBERNETES`?